keycloak/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak
+
+Keycloak is an Open Source Identity and Access Management solution for modern Applications and Services, for more details about it vistit [Keycloak oficial repository](https://github.com/keycloak/keycloak)
```

Comment:
vistit => visit

Reply:
Thanks for the observation, it was fixed.